packages/storybook/src/primitives/Checkbox/Checkbox.stories.tsx

```diff
@@ -19,3 +19,59 @@ export function initial() {
     </>
   )
 }
+
+export function Indeterminate() {
+  const [checkedItems, setCheckedItems] = React.useState<{
+    [key: string]: boolean
+  }>({
+    option1: false,
+    option2: false,
+  })
+
+  const allChecked = Object.keys(checkedItems)
```

Comment:
💡 `Object.keys()` cannot return null so perhaps the `?` is redundant. and we can also write it like:

`const allChecked = Object.values(checkedItems).every(Boolean)`

Reply:
Ah yes, on the Boolean. I'll update that real quick. Thanks!